siliconcompiler/apps/sc_dashboard.py

```diff
@@ -54,12 +55,24 @@ def main():
 
     graph_chips = []
     if switches['graph_cfg']:
-        for i, file_path in enumerate(switches['graph_cfg']):
+        for name_and_file_path in switches['graph_cfg']:
+            args = len(name_and_file_path)
```

Comment:
I think len will always be >1 since its a string you are parsing, you may need to split the string and then check

Reply:
On my end, name_and_file_path is a list. where the elements are the strings separated by spaces. Maybe you should test this as well, there seems to be some difference between our systems.